cloudwash/cli.py

```diff
@@ -52,13 +52,14 @@ def gce(ctx, vms, discs, nics, _all):
 @cleanup_providers.command(help='Cleanup Azure provider')
 @common_options
 @click.option('--pips', is_flag=True, help='Remove only PiPs from the provider')
+@click.option('--all_if_older', '_all_if_older', is_flag=True, help='Remove provider group if all resources are older than SLA')
```

Comment:
```suggestion
@click.option('--all_if_older', '_all_if_older', is_flag=True, help='Remove resource group only if all resources are older than SLA')
```

I think we should also rename VM_SLA in settings.yaml to enable for all resources.

Reply:
I would argue against it as this way it's name is more generic and makes more sense with `--all_rg' flag, where this limit is used against all resources and not just VMs. As for other uses it's purpose is described [here].(https://github.com/RedHatQE/cloudwash/blob/cc40621136bbdd502e887cc02c6de5a486ef80d9/settings.yaml.template#L4 )

Reply:
@laDok8 Ah, wanted to say something else and said something else! Please forget !